### DISCUSSION TITLE
Scribble Backend Layouts

### DISCUSSION THREAD

**Discussion by thelf** [Webmasters-Europe/codemeetup-forum#107] [General]
Ziel ist das Entwerfen von mehreren Backend Layouts welche die verschiedenen Zustände abbilden (z.B. wie sieht das Layout für die Administratoren aus wenn sie Benutzer verwalten, Artikel im Forum verwalten usw.)

- Ideen sammlen welche Elemente benötigt werden um das Forum zu strukturieren und bedienen zu können
- Elemente in ein Scribble einsetzen (Buttons, Input Felder, Text Bereiche usw.)
- Scribbeln von verschiedenen Layouts aus den einzelnen Elementen
- Umsetzung der Layouts als klickbarer HTML Prototyp 

Die Layouts sollten alle responsive sein um auch auf mobilen Endgeräten ohne Einschränkungen benutzbar zu sein.

Vorschlag für Open Source Software für die Erstellung von Scribbles (Windows, Mac und Ubuntu wird unterstützt) https://pencil.evolus.vn/

**Comment by milack68** [comment]
Gute Idee, ich werde mir das Tool anschauen. Habe ich bisher nicht gekannt.
Ich arbeite bisher immer mit [draw.io](https://app.diagrams.net/).

**Comment by thelf** [comment]
Ok, draw.io kenne ich bisher noch nicht, ich denke generell ist es gut vorher ein wenig zu scribbeln bevor es an die Umsetzung geht.
